mgmt/utils/MgmtSocket.cc

```diff
@@ -23,7 +23,7 @@
 
 #include "ts/ink_platform.h"
 #include "ts/ink_assert.h"
-#include <ts/ink_cap.cc>
+#include "ts/ink_cap.h"
```

Comment:
Are you perhaps removing @SolidWallOfCode 's work around for some other exe build where MgmtSocket.o is linked in but ink_cap.o is not?

Reply:
No, this was a typo, it should always have been `.h`.